alfalfa_web/server/api-v2.js

```diff
@@ -164,362 +252,402 @@ router.get("/sites/:siteId", async (req, res) => {
  *             example:
  *               time: 2022-06-15 00:00:00
  */
-router.get("/sites/:siteId/time", async (req, res) => {
-  const { siteId } = req.params;
-
-  const error = validate(
-    { siteId },
-    {
-      siteId: "required|uuid"
-    }
-  );
-  if (error) return res.status(400).json({ error });
-
+router.get("/runs/:runId/time", async (req, res, next) => {
   api
-    .getSiteTime(siteId)
-    .then((time) => res.json({ time }))
-    .catch(() => res.sendStatus(500));
+    .getRunTime(req.run)
+    .then((time) => res.json({ payload: { time } }))
+    .catch(next);
 });
 
 /**
  * @openapi
- * /sites/{siteId}/points:
+ * /runs/{runId}/points:
  *   get:
- *     operationId: List points
- *     description: Return list of points for a site
+ *     summary: Get the metadata for all points of an associated run
  *     tags:
- *       - Site
+ *       - Run
  *     parameters:
- *       - $ref: '#/components/parameters/SiteID'
+ *       - $ref: '#/components/parameters/runId'
  *     responses:
- *       200:
- *         description: Points response
+ *       '200':
+ *         description: OK
  *         content:
  *           application/json:
  *             schema:
- *               type: object
+ *               type: array
+ *               items:
+ *                 $ref: '#/components/schemas/pointMetadata'
+ */
+router.get("/runs/:runId/points", (req, res, next) => {
+  api
+    .getPointsByRun(req.run)
+    .then((points) => {
+      return points.map(api.formatPoint);
```

Comment:
All db objects now have an accompanying `format` function which is used to present them to the user.